sdcm/utils/issues_by_keyword/issue_by_keyword.yaml

```diff
@@ -0,0 +1,12 @@
+REACTOR_STALLED: # event type
+  - keyword: row_cache::upgrade_entry
+    issue: https://github.com/scylladb/scylladb/issues/2577
+
+  - keyword: partition_entry::upgrade
+    issue: https://github.com/scylladb/scylladb/issues/2577
+
+  - keyword: utils::uleb64_express_encode
+    issue: https://github.com/scylladb/scylladb/issues/8828
+
+  - keyword: logalloc::region_impl::object_descriptor::encode stalls
+    issue: https://github.com/scylladb/scylladb/issues/8828
```

Comment:
Aren't these keywords too generic and we start to mislabel events with wrong issues?
Did you discuss these keywords with developers?

When investigating I'm afraid we'll start to ignore all events labelled as known issue. Idea to prevent it would be to change severity based on issue status - if issue is closed severity should be increased to error (to possibly reopen it or create new one).

Reply:
I received those keywords from Avi

Reply:
> Aren't these keywords too generic and we start to mislabel events with wrong issues? Did you discuss these keywords with developers?
> 
> When investigating I'm afraid we'll start to ignore all events labelled as known issue. Idea to prevent it would be to change severity based on issue status - if issue is closed severity should be increased to error (to possibly reopen it or create new one).

Hmm, this is the good idea

Reply:
SkipPerIssue can be used here

Reply:
@juliayakovlev let's add it as a follow up

Reply:
@fruch do I remember correct that we need to add "skip..<version>" label to the issue?

Reply:
'sct-{branch_version}-skip'